<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listBuilderWindow</class>
 <widget class="QMainWindow" name="listBuilderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>850</height>
   </size>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Boarding Actions - List Building</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="factionLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>950</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saveListButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>EXPORT LIST as PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeListButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>CLOSE APP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToListButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To List</string>
    </property>
   </widget>
   <widget class="QComboBox" name="unitChoiceComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>381</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="listStatusBarLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>700</y>
      <width>950</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="unitListTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>661</width>
      <height>570</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>Unit Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit Dependencies</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit Sizes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit Cost</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="clearListButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR LIST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeSelectedUnitButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVE UNIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="validateListButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>VALIDATE LIST</string>
    </property>
   </widget>
   <widget class="QLabel" name="validationLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>750</y>
      <width>950</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCloseList"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCloseList">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
